types/iopipe/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for iopipe 1.6
+// Project: https://github.com/iopipe/the-meta-package#readme
+// Definitions by: Javon Harper <https://github.com/javonharper>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface LibraryConfig {
```

Comment:
This package is currently global, but it should be a module that re-exports `@iopipe/core`:

```ts
export * as iopipe
```

and probably nothing else. Not sure where would be best to put the interface. Probably one of the dependency packages.

Reply:
Thanks, I slimmed down the definitions to only cover @iopipe/iopipe.